content/_index.md

```diff
@@ -1,44 +1,34 @@
 ---
 title: Home
 ---
-{{< home/section title="What?" iconclass="fas fa-2x fa-power-off">}}
-GitBOM is neither [git](/glossary/git) nor an [SBOM](/resources/glossay/#sbom).
-<!-- 
-GitBOM is an ultra simple scheme for [primary build tools](/resources/glossay/#primary-build-tools) to embed into the artifact being built a
-short simple verifiable reference to complete artifact tree of that artifact with absolute precision all the way down to the source code. -->
+{{< home/section title="What is GitBOM?" iconclass="fas fa-2x fa-power-off">}}
+GitBOM is creative re-purposing of the [git](https://en.wikipedia.org/wiki/Git) [dag](https://en.wikipedia.org/wiki/Directed_acyclic_graph) to enable *A*utomatic and *V*erifiable *A*rtifact *R*esolution across today's language-heterogeneous open source supply chains. Contrary to the name's appearance, GitBOM is neither [git](/glossary/git) nor an [SBOM](/glossary/sbom).
 
 GitBOM is a minimalistic scheme for [build tools](/glossary/build_tool) to:
 
-1. Build a compact [artifact tree](/glossary/artifact_tree), tracking every source code file incorporated into each built [artifact](/glossary/artifact).
-2. Embed an id for that artifact tree into the artifact being built.
-3. In a language-heterogenous environment, across packaging formats, with zero developer effort, involvement, or awareness.
+1. Build a compact [artifact tree](/glossary/artifact_tree), tracking both source and object files incorporated in the generation of a built [artifact](/glossary/artifact).
+2. Embed a unique content-addressable reference for that [artifact tree](/glossary/artifact_tree) into the [artifact](/glossary/artifact) at build time.
+3. And do this consistently, in a language-heterogenous environment, across packaging formats, with zero developer effort, involvement, or awareness.
 
-- GitBOM is not an SBOM. It is designed to complement SBOMs, such as [SPDX](https://spdx.dev/), [CycloneDX](https://cyclonedx.org/), or [SWID](https://nvd.nist.gov/products/swid).
-- GitBOM is not a version control system. It is designed to co-exist with, but not require, them.
+GitBOM applies the [Unix Philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) of "do one thing, and do it well": GitBOM constructs [artifact trees](/glossary/artifact_tree). GitBOM *is not* a replacement for existing SBOM tools or standards, nor is it a version control system, nor a signing scheme. It is compatible with, and augments, each of these classes of tools.
```

Comment:
This seems to push two points together:

1. Not an SBOM
2. Do one thing and do it well.

I feel like we should be clear and crisp on both points, and they get a bit muddled pushed together.

Reply:
Hmm... ok.

What if the second sentence, "gitbom is not a replacement ..." were pulled into an in-line note? That would visually make these points distinct, while also separating the negation into a callout, away from the four prior affirmative statements.

Reply:
I like the inline note idea.  It draws *strong* attention to the 'not a replacement for existing SBOM tools' which is something we want to be very very very clear :)

Reply:
I believe I addressed these comments with the changes proposed in https://github.com/git-bom/site/pull/10/files